packages/create-cloudflare/src/helpers/command.ts

```diff
@@ -72,7 +70,7 @@ export const runCommand = async (
 
 			let output = ``;
 
-			if (opts?.captureOutput ?? squelch) {
+			if (opts?.captureOutput ?? opts.silent) {
```

Comment:
I don't think `opts` can be `undefined`, can it?

```suggestion
			if (opts.captureOutput ?? opts.silent) {
```